org.civicrm.reqangular/src/common/angular-date/src/services/DateFormatFactory.js

```diff
@@ -0,0 +1,36 @@
+/**
+ * This factory provides access to date format from CRM settings
+ * @param $q
+ * @returns {{dateFormat: null, getDateFormat: getDateFormat}}
+ * @constructor
+ */
+function DateFormatFactory($q) {
+    return {
+        /**
+         * keeps information about date format
+         */
+        dateFormat: null,
+        /**
+         * @description Fetches date format setting from TODO where?
```

Comment:
the `@description` tag is not necessary if the description is the first thing in the comment section